src/data-card-advanced.tsx

```diff
@@ -0,0 +1,463 @@
+
+interface DataCardAdvancedProps {
+    cardOnPress?: () => void;
+
+    stackingGroup?: RendersNullableElement<typeof StackingGroup>;
+    headline?: string | RendersNullableElement<typeof Tag>;
+    pretitle?: string;
+    pretitleLinesMax?: number;
+    pretitleAs?: textAs;
+    title?: string;
+    titleLinesMax?: number;
+    titleAs?: textAs;
+    subtitle?: string;
+    subtitleLinesMax?: number;
+    description?: string;
+    descriptionLinesMax?: number;
+
+    slots?: Array<RendersNullableElement<slotsTypeof>>;
+    small?: boolean;
+
+    button?: RendersNullableElement<typeof ButtonPrimary>;
+    footerImage?: RendersNullableElement<typeof Image>;
+    footerText?: string;
+    footerTextLinesMax?: number;
+    buttonLink?: RendersNullableElement<typeof ButtonLink>;
+
+    dataAttributes?: DataAttributes;
+    actions?: Array<CardAction>;
+    'card-aria-label'?: string;
+    'touchable-aria-label'?: string;
+    onClose?: () => void;
+}
+
+const DataCardAdvanced = React.forwardRef<HTMLDivElement, DataCardAdvancedProps>(
+    (
+        {
+            cardOnPress,
+
+            stackingGroup,
+            headline,
+            pretitle,
+            pretitleAs,
+            pretitleLinesMax,
+            title,
+            titleAs,
+            titleLinesMax,
+            subtitle,
+            subtitleLinesMax,
+            description,
+            descriptionLinesMax,
+
+            slots,
+            small,
+
+            button,
+            footerImage,
+            footerText,
+            footerTextLinesMax,
+            buttonLink,
+
+            dataAttributes,
+            actions,
+            'touchable-aria-label': touchableAriaLabel,
+            'card-aria-label': cardAriaLabel,
+            onClose,
+        },
+        ref
+    ) => {
+        const finalActions = useTopActions(actions, onClose);
+        const hasAcations = finalActions?.length > 0;
+        const hasStackingGroup = !!stackingGroup;
+
+        const hasFooter = !!button || !!footerImage || !!footerText || !!buttonLink;
+        const footerProps = {button, footerImage, footerText, footerTextLinesMax, buttonLink};
+
+        const slotSpaceSize = small ? 8 : 24;
+
+        const topActionsStylesWithIcon = {position: 'absolute', top: 8, right: 8, zIndex: 2} as const;
+        const cardContentStyle = sprinkles({
+            display: 'flex',
+            paddingBottom: hasFooter || slots ? 24 : 0,
+        });
+
+        return (
+            <section aria-label={cardAriaLabel} style={{height: '100%', position: 'relative'}}>
+                <Boxed
+                    className={styles.boxed}
+                    dataAttributes={{'component-name': 'DataCard', ...dataAttributes}}
+                    ref={ref}
+                    width="100%"
+                    height="100%"
+                >
+                    <div className={styles.dataCard}>
+                        {cardOnPress && (
+                            <a
```

Comment:
for accessibility reasons due to a conflict that previously occurred with Touchable

Reply:
The reason for the conflict is to get the buttons working inside the touchable.

in the case where you have:
// eslint-disable-next-line jsx-a11y/no-static-element-interactions

The Div wrapped around the buttons serves to prevent the card's onClick event from being triggered when the buttons event is, because of this onClick, the div did not accept focusable (such as buttons, inputs, etc.), and this generated conflict.

without this eslint-disable, the error appears in the code of:
Static HTML elements with event handlers require a role.eslintjsx-a11y/no-static-element-interactions

however, when using a role, aria-hidden, etc, it still triggered the audit-a11y accessibility error in PR

I just managed to 'resolve' this error by using this eslint-disable because there's no way to remove the div around the button, and there's no way to put the button outside of it, or by an aria-hidden, role, aria-disable, etc.